config/config.go

```diff
@@ -190,6 +190,9 @@ type ServerConfig struct {
 	// HTTP clients to be used for API connections
 	HTTPClient          *http.Client
 	HTTPClientWithRetry *http.Client
+
+	// Data directory used for system stats
+	DataDirectory string
```

Comment:
Not sure if it's a great idea to put this in `ServerConfig`, but the proxy clients being here seem like a reasonable precedent

Reply:
Yeah, I guess it couldn't hurt to allow an override. I don't think CollectionStatus makes sense given the other fields in that struct.